CMakeLists.txt

```diff
@@ -129,7 +148,13 @@ setup_conan()
 
 # If we do not set them with a space CMake fails afterwards if nothing is set for this vars!
 set(AER_LINKER_FLAGS " ")
-set(AER_COMPILER_FLAGS " ")
+
+# This limit is hit building the Python plugin for CUDA
+if(APPLE OR UNIX)
+	set(AER_COMPILER_FLAGS " ")
+elseif(MSVC)
+	set(AER_COMPILER_FLAGS " /bigobj")
+endif()
```

Comment:
> I think it is safe to remove this now and leave the original code, as some default windows flags are now set above (line 131)

I checked this on a Windows build and it is still necessary for CUDA builds.  Presumably because it is not C++ as far as CMake is concerned.  (The addition of /bigobj is for C++ files only.)   I have a follow-up branch that tweaks the Pybind11 header file so that WIndows builds work, and I reinstalled that change there for now.